gopls/doc/subl.md

```diff
@@ -1,81 +1,77 @@
 # Sublime Text
 
-Use the [LSP] package. After installing it using Package Control, do the following:
+Setting up Sublime Text for Golang development.
 
-* Open the **Command Palette**
-* Find and run the command **LSP: Enable Language Server Globally**
-* Select the **gopls** item. Be careful not to select the similarly named *golsp* by mistake.
+## Installation
 
-Finally, you should familiarise yourself with the LSP package's *Settings* and *Key Bindings*. Find them under the menu item **Preferences > Package Settings > LSP**.
+Make sure Go is installed and available in your `PATH`. The [Go Documentation][golang-installation] get Go up and running quickly.
+You can verify that Go is properly installed and available by typing `go help` in a terminal. The command should print a help text instead of an error message.
 
-## Examples
-Minimal global LSP settings, that assume **gopls** and **go** appear on the PATH seen by Sublime Text:<br>
-```
+Use [Package Control] to install the following plugins:
+
+- [LSP] provides language server support in Sublime Text
+- [LSP-gopls] the helper plugin for gopls
+- (optionally) [Gomod] and [Golang Build] for gomod support and a Go build system
+
+gopls is automatically installed and activated when a `.go` file is opened.
+
+## Configuration
+
+Here are some ways to configure the package and the language server. See [the documentation][gopls-settings] for all available settings.
+
+### Global configuration
+- Configure the LSP plugin by navigating to `Preferences > Package Settings > LSP > Settings` or by executing the `Preferences: LSP Settings` command in the command palette.
+- Configure gopls by navigating to `Preferences > Package Settings > LSP > Servers > LSP-gopls` or by executing the `Preferences: LSP-gopls Settings` command in the command palette.
+
+### Project-specific configuration
+From the command palette run `Project: Edit Project` and add your settings in:
+
+```js
 {
-    "clients": {
-        "gopls": {
-             "enabled": true,
-         }
+  "settings": {
+    "LSP": {
+      "LSP-gopls": {
```

Comment:
This is incorrect, the key is `"gopls"`: https://github.com/sublimelsp/LSP-gopls/blob/d349280d2b29e5242226669d42f39b47e69db738/sublime-package.json#L525

Reply:
Fixed. Thanks!